user_manual/groupware/absence.rst

```diff
@@ -0,0 +1,14 @@
+.. _groupware-absence:
+
+==============================
+Setting out-of-office messages
+==============================
+
+If you are absent for vacation, sick leave or similar, you can add an out-of-office message in the **Absence** section of the **Availability** settings.
+
+The interface asks for the time of absence, a short and a long message. This data is used for the following purposes:
+
+1) Your user status will change to the short message when your absence starts and reset when it ends.
+2) And event with status *busy* will be created in your personal calendar. This allows others to see that you are not available when they use the free/busy feature.
+3) If enabled, the Mail app will apply an autoresponder.
```

Comment:
Can you elaborate that the short message will be the subject and the long message the mail body here? (IIRC from the email I have received)

Reply:
That is something I wanted to clarify, hence the vague documentation. I'm not sure if short message should even be used as subject.

Reply:
https://github.com/nextcloud/mail/issues/8925#issuecomment-1864599786 let's see ;)

Reply:
https://github.com/nextcloud/mail/issues/9219

Reply:
This is still a bit vague. We only use the long message as the body after https://github.com/nextcloud/mail/pull/9228.